### DISCUSSION TITLE
Advice on normalizing across multiple runs?

### DISCUSSION THREAD

**Discussion by whitneyt1** [angelolab/toffy#301] [Q&A]
Hello! 

We're looking to analyze a number of runs that were run separately, thus have multiple run_names/folders etc. Is there any way to normalize all of these runs together instead of one by one? 

Thanks!

**Comment by ngreenwald** [comment]
I would recommend putting the cells in the normalization notebook within a for loop that iterates over each of your run folders. Because of the how the normalization values are calculated, it needs to be done separately for each run. Once you have normalized the runs separately, they can combined together into one single directory for analysis purposes with notebook 5
